source/draw/DrawState.ooc

```diff
@@ -43,11 +45,25 @@ DrawState: cover {
 		this
 	}
 	// Local region
-	setDestination: func (destination: IntBox2D) -> This {
-		this destination = destination
-		this
+	// Precondition: target exists
```

Comment:
Or really not be here at all. Obviously `target` must exist if `this target size` is to make any sense.